### ISSUE TITLE
Platypus: missing `lang` attribute

### ISSUE THREAD

**Issue by eddybrando** [Qiskit/qiskit.org#2066]
According to an Axe a11y audit performed on `/course/ch-prerequisites/environment-setup-guide-to-work-with-qiskit-textbook`, the `<html>` element is missing a `lang` attribute.

---

https://dequeuniversity.com/rules/axe/3.5/html-has-lang

---

**User impact:** Serious
**Disabilities affected:** Blind, Deafblind, Cognitive
**Requirement(s):** WCAG 2.0 (A): MUST
**WCAG Success Criteria:** 3.1.1 Language of Page
